python/pyspark/sql/functions.py

```diff
@@ -2058,7 +2058,7 @@ def __init__(self, func, returnType, name=None, vectorized=False):
         self._name = name or (
             func.__name__ if hasattr(func, '__name__')
             else func.__class__.__name__)
-        self._vectorized = vectorized
+        self.vectorized = vectorized
```

Comment:
Oh sorry, I should have been more clear.  This should stay `self._vectorized` since it is a private variable to the class, it's only `wrapped.vectorized` (which you already changed below), isn't being used as private so shouldn't have an underscore.

Reply:
I kind of dislike the inconsistency between `UserDefinedFunction` and its wrapped function. I think they are just the same thing except for the wrapped function has doc string. For ease of mind, I think we should make them either both private or public.

Reply:
Are we ok with having `vectorized` being public field? I am fine with both public or private but I do think the fields of the function returned by `UserDefinedFuncion_wrapped()` should have the same field names as `UserDefinedFunction` to avoid confusion.